### PR TITLE
removed check mode from doc fragment and placed it in every module. a…

### PR DESCRIPTION
**Pull request by ar-india**
…lso removed .plugin_cache.yml

Signed-off-by: ar-india <alokranjan2005@gmail.com>

### CODE REVIEW

**Comment by ar-india** [comment]
this is being done because as per ansible reviewers requirement , info module requires check mode to be supported and hence we can no longer keep the string "check mode not supported" in doc_fragments.

**Review by suneeth51 — APPROVED** [review]
lgtm..

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_access_control_record.py [hunk 1]
```diff
@@ -66,6 +66,8 @@
 extends_documentation_fragment: hpe.nimble.hpe_nimble
 short_description: Manage the HPE Nimble Storage access control records
 version_added: "1.0.0"
+notes:
+  -  check_mode is not supported.
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_access_control_record.py]: Everyone has double space between "-" and "check_mode".

Also, is backticks supported here, as it's a parameter.

**Review by datamattsson — CHANGES_REQUESTED** [review]

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py [hunk 2]
```diff
@@ -208,7 +208,7 @@
 short_description: Manage the HPE Nimble Storage volume collections
 version_added: "1.0.0"
 notes:
-  -  check_mode is not supported.
+  - This module does not supports C(check_mode).
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py]: This module does not support C(check_mode). (no ending "s" on support). I've not seen the C() annotation before but I guess that's ansible specific?

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_access_control_record.py (hunk 1 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_access_control_record.py]: ok changed to "- This module does not supports C(check_mode)."

**Review by datamattsson — CHANGES_REQUESTED** [review]

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py (hunk 2 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py]: yes that is ansible specific.. that's fine

Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py]: ok will remove 's'

**Review by datamattsson — APPROVED** [review]
LGTM

**Pull request merged**